ui/components/app/signature-request/signature-request.component.js

```diff
@@ -174,7 +175,7 @@ export default class SignatureRequest extends PureComponent {
       : null;
 
     return (
-      <div className="signature-request page-container">
+      <div className="signature-request">
```

Comment:
curious why the `page-container` class was removed here. It seems that removing this will cause the content to be left-aligned now as opposed to centered in full-page mode

before w/ page-container: 
<img width="1503" alt="Screen Shot 2022-12-02 at 04 40 22" src="https://user-images.githubusercontent.com/20778143/205164806-7e7fd89a-3f82-4a24-83d1-1a6f4d33ded3.png">

after:
<img width="1508" alt="Screen Shot 2022-12-02 at 04 40 47" src="https://user-images.githubusercontent.com/20778143/205164968-0b8ff065-076b-4b23-8f90-bc6bd25d6973.png">

Reply:
Hi @digiwand 
I remove page-container class to fix the size of the signature screen.

I pushed changes that resolve the storybook broke issue. Also, the content will be centered.

Reply:
Gotchya. Nice fix and thanks for the updates!